plugin/admin/options/option-types/class-selectfield.php

```diff
@@ -37,28 +37,43 @@ public function register() {
 	 * @return int Sanitized value.
 	 */
 	public function sanitize( $value ) {
-		if ( 'pixels' === $value ) {
-			return 'pixels';
-		}
-
-		if ( 'percentage' === $value ) {
-			return 'percentage';
-		}
-
-		return $this->default_value;
+		return esc_html( $value );
```

Comment:
There is no big sense to validate values returned by checkboxes. It will be always valid because users can't modify its values.

Reply:
Hmm, I was thinking about the users not being able to deliberately break the plugin or insert XSS or something... But true, this would protect against XSS and anyway, this should only be editable by administrators....